proto/sensei.proto

```diff
@@ -115,6 +115,8 @@ message CreateNodeRequest {
 message CreateNodeResponse {
     string pubkey = 1;
     string macaroon = 2;
+    string listen_addr = 8;
+    int32 listen_port = 9;
```

Comment:
can we also add the node id. 

also i'm not actually familiar with this numbering or how it works but I think maybe it needs to/ should go in order?

Reply:
can do!